server/docker/mtt_db.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+psql --command "CREATE USER mtt WITH LOGIN PASSWORD 'mttuser' CREATEDB CREATEROLE;" && \
+psql --command "CREATE USER mtt_viewer WITH LOGIN PASSWORD 'mttviewer';" && \
+psql --command "CREATE DATABASE mtt WITH OWNER mtt;" && \
+psql -U mtt mtt -f /opt/mtt/server/sql/schemas-v3.sql && \
+psql -U mtt mtt -f /opt/mtt/server/sql/summary/summary_tables.sql && \
+psql -U mtt mtt -f /opt/mtt/server/sql/summary/summary_trigger.sql && \
+psql -U mtt mtt -f /opt/mtt/server/sql/schemas-stats.sql && \
+psql -U mtt mtt -f /opt/mtt/server/sql/schemas-reporter.sql && \
+psql -U mtt mtt -f /opt/mtt/server/sql/schemas-indexes.sql
```

Comment:
For the v3 database you also need to generate the partition tables for the given year.

Something like this (might need to some tweaking) will add all of the partition tables for 2017 to the db. Note the `XX` is a special marker for the scripts to indicate all months.
```shell
YEAR=2017
/opt/mtt/server/sql/support/create-partitions-mpi-install.pl $YEAR XX > /tmp/$YEAR-mpi-install.sql
/opt/mtt/server/sql/support/create-partitions-test-build.pl $YEAR XX > /tmp/$YEAR-test-build.sql
/opt/mtt/server/sql/support/create-partitions-test-run.pl $YEAR XX > /tmp/$YEAR-test-run.sql
/opt/mtt/server/sql/support/create-partition-indexes.pl $YEAR XX > /tmp/$YEAR-indexes.sql
/opt/mtt/server/sql/summary/create-partition-triggers.pl $YEAR XX > /tmp/$YEAR-triggers.sql

psql -U mtt mtt -f /tmp/$YEAR-mpi-install.sql
psql -U mtt mtt -f /tmp/$YEAR-test-build.sql
psql -U mtt mtt -f /tmp/$YEAR-test-run.sql
psql -U mtt mtt -f /tmp/$YEAR-indexes.sql
psql -U mtt mtt -f /tmp/$YEAR-triggers.sql
```

For the partition tables, this is something we have to do every year.

You might be able to use the high level wrapper script:
```shell
/opt/mtt/server/sql/support/yearly-table-update.pl 2017
```